dashboards/grafana-dashboard-kruize-metrics.configmap.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 data:
-  Kruize_Metrics.json: |-
+  kruize-metrics.json: |
```

Comment:
Adding comment for line 5 here as it is related to already committed code.
My [comment](https://github.com/RedHatInsights/ros-ocp-backend/pull/134/files#r1381547766) from ros ocp side is applicable here as well:

Instead of __inputs variables, please make use of templating variable (i.e datasource).
Refer - https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/app-sre/monitoring.md#setting-the-correct-datasource